### PR TITLE
Merge meta.yaml

### PR DESCRIPTION
**Pull request by GeekArthur**
Signed-off-by: jingfu wang <jingfwan@redhat.com>

This PR merges meta.yaml to devfile.yaml

Related issue: https://github.com/devfile/api/issues/222

### CODE REVIEW

**Comment by GeekArthur** [comment]
This PR should be merged after this PR https://github.com/devfile/registry-support/pull/28 is merged

File: stacks/java-quarkus/devfile.yaml [hunk 1]
```diff
@@ -2,7 +2,11 @@ schemaVersion: 2.0.0
 metadata:
   name: java-quarkus
   version: 1.1.0
-  website: https://quarkus.io
```
Comment (johnmcollier) [inline stacks/java-quarkus/devfile.yaml:5]: Do we want to remove the project field from the devfile?

**Review by johnmcollier — COMMENTED** [review]
Left one small comment, otherwise looks good

File: stacks/java-quarkus/devfile.yaml (hunk 1 above)
Comment (GeekArthur) [reply stacks/java-quarkus/devfile.yaml:5]: I checked the metadata field of API reference: https://docs.devfile.io/devfile/2.1.0/user-guide/api-reference.html and found the website field is not defined under metadata field, that's why I removed that, but I am open to add it back to both this devfile.yaml and API reference as I am gonna create another PR for API reference to add some fields (eg. projectType, language) under metadata.

Comment (GeekArthur) [reply stacks/java-quarkus/devfile.yaml:5]: Do you think we should add the website field back to both this devfile.yaml and API reference? Currently in our registry repo, only java-quarkus has the website field under metadata.

**Review by johnmcollier — APPROVED** [review]

**Review by johnmcollier — APPROVED** [review]

**Pull request merged**